cmd/zpool/zpool_main.c

```diff
@@ -5522,10 +5524,14 @@ zpool_do_trim(int argc, char **argv)
 
 	cb.cb_start = B_TRUE;
 	cb.cb_rate = 0;
+	cb.cb_fulltrim = B_FALSE;
 
 	/* check options */
-	while ((c = getopt(argc, argv, "sr:")) != -1) {
+	while ((c = getopt(argc, argv, "fsr:")) != -1) {
```

Comment:
The get_usage() function needs to be updated to include the `-f` option so it's shown when running `zpool trim --help`.